go-controller/cmd/ovnkube/ovnkube.go

```diff
@@ -20,6 +20,7 @@ import (
 
 	"github.com/ovn-org/ovn-kubernetes/go-controller/pkg/config"
 	"github.com/ovn-org/ovn-kubernetes/go-controller/pkg/factory"
+	"github.com/ovn-org/ovn-kubernetes/go-controller/pkg/metric"
```

Comment:
@girishmg can we make this package `metrics` (eg plural)?

Reply:
@dcbw will do